lib/miq_automation_engine/service_models/miq_ae_service_container_deployment_node.rb

```diff
@@ -0,0 +1,15 @@
+module MiqAeMethodService
+  class MiqAeServiceContainerDeploymentNode < MiqAeServiceModelBase
+    expose :vm_id
+    expose :is_tagged_with?
```

Comment:
The tagging we removed was the one for the provisioning, this one isn't related to it, in this case we use tags to know what roles a container deployment node has.